waspc/data/Generator/templates/server/tsconfig.json

```diff
@@ -3,6 +3,15 @@
   "extends": "@tsconfig/node{= majorNodeVersion =}/tsconfig.json",
   "compilerOptions": {
     // Overriding this until we implement more complete TypeScript support.
+    // Filip: begin client file hacks
+    // We need this to make server work with copied client files (we copy everything)
+    "jsx": "preserve",
+    "lib": [
+      "esnext",
+      "dom",
+      "DOM.Iterable"
+    ],
+    // Filip: end client file hacks
```

Comment:
Todo: Revisit this, probably still necessary because we're importing user files, but we should try to find a way to avoid it.